frontend/src/components/common/Select/SelectedOption.tsx

```diff
@@ -0,0 +1,131 @@
+import styled from '@emotion/styled';
+import { IconButton } from '../../common/IconButton/IconButton';
+
+export interface SelectedItem {
+  value: string;
+  label: string;
+}
+
+interface SelectedOptionProps {
+  selectedItems: SelectedItem[];
+  onRemove: (value: string) => void;
+}
+
+// Define the SVG content for the DRAG icon
+const DragIconSVG = (
+  <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M15.75 15.5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0Zm0-7a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0Zm7 7a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0Zm0-7a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0Zm7 7a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0Zm0-7a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0Z"
+      fill="currentColor"
+    />
+  </svg>
+);
+
+// Define the SVG content for the CLOSE icon
+const CloseIconSVG = (
+  <svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="m4.21 4.387.083-.094a1 1 0 0 1 1.32-.083l.094.083L12 10.585l6.293-6.292a1 1 0 1 1 1.414 1.414L13.415 12l6.292 6.293a1 1 0 0 1 .083 1.32l-.083.094a1 1 0 0 1-1.32.083l-.094-.083L12 13.415l-6.293 6.292a1 1 0 0 1-1.414-1.414L10.585 12 4.293 5.707a1 1 0 0 1-.083-1.32l.083-.094-.083.094Z"
+      fill="currentColor"
+    />
+  </svg>
+);
+
+export function SelectedOption({ selectedItems, onRemove }: SelectedOptionProps) {
+  return (
+    <StyledContainer>
+      {selectedItems.map((item) => (
+        <SelectedItemWrapper key={item.value}>
+          <ItemContent>
+            <StyledPlaceholderIconButton>{DragIconSVG}</StyledPlaceholderIconButton>
```

Comment:
This should be before `ItemContent`. Reserve `ItemContent` for the label (and an optional label caption - see the svelte version)